config/resources/test_extensive_config.json

```diff
@@ -0,0 +1,126 @@
+{
+  "aclTokenSecret": {
+    "provider": "secrets-manager",
+    "configuration": {
+      "prefix": "abc123",
+      "consulClientTokenSecretArn": "some-long-arn"
+    }
+  },
+  "mesh": {
+    "bootstrapDir": "/consul/",
+    "healthSyncContainers": ["frontend"],
+    "service": {
+      "name": "frontend",
+      "tags": ["frontend"],
+      "port": 8080,
+      "address": "127.0.0.1",
+      "socketPath": "/path/to/socket",
+      "taggedAddresses": {
+        "lan": {
+          "address": "192.168.1.5",
+          "port": 8080
+        }
+      },
+      "enableTagOverride": true,
+      "meta": {
+        "env": "test",
+        "version": "x.y.z"
+      },
+      "weights": {
+        "passing": 6,
+        "warning": 5
+      },
+      "checks": [
+        {
+          "checkId": "frontend-http",
+          "name": "frontend-http",
+          "http": "http://localhost:8080/health",
+          "method": "POST",
+          "body": "{\"method\": \"health\"}",
+          "notes": "Health check for the frontend service",
+          "header": {
+            "Content-Type": ["application/json"]
+          },
+          "interval": "30s",
+          "timeout": "10s",
+          "successBeforePassing": 3,
+          "failuresBeforeCritical": 4,
+          "deregisterCriticalServiceAfter": "60m"
```

Comment:
Not sure if we should offer this since the health check work handles this so this might throw a wrench into things.